### DISCUSSION TITLE
Add example showing webpage that reacts to database updates

### DISCUSSION THREAD

**Discussion by jcyrio** [reflex-dev/reflex#149] [General]
It would be helpful to have an example that shows page data that updates automatically when a database is updated.

Also, it might be helpful to show an example that shows a page being updated upon receiving data via MQTT, since this is a popular IOT scenario, and it seems that pynecone would be perfect for IOT apps.

**Comment by picklelo** [comment]
Great idea! Yes we definitely want an example app of some sort of live updating dashboard, seems like a common type of app.
